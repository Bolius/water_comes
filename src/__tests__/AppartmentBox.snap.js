import React from "react";
import { create } from "react-test-renderer";
import ApartmentBox from "../components/apartment-box.js";

describe("Box in case of appartment", () => {
  test("Matches the snapshot", () => {
    const appBox = create(<ApartmentBox />).toJSON();
    expect(appBox).toMatchSnapshot();
  });
});
