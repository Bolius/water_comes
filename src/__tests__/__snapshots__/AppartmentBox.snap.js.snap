// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box in case of appartment Matches the snapshot 1`] = `
<div
  className="appartmentbox"
>
  <h3>
    OBS: Din adresse viser, at du bor i lejlighed.
  </h3>
  <p>
    Derfor er risikoberegningen kun gældende for kælderen. Husk stadig at lukke dine vinduer og evt. altandør.
  </p>
</div>
`;
